tests/integration/test.basics.js

```diff
@@ -1043,6 +1043,10 @@ adapters.forEach(function (adapter) {
       db.type().should.be.a('string');
     });
 
+    it('#4788 db.type() is synchronous', function () {
+      new PouchDB(dbs.name).type.should.be.a('function');
```

Comment:
No. You don't need to repeat it twice. I am not sure why.
I checked for failure in browser test like discussed.